pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -191,8 +201,21 @@ func testSendDataSet(t *testing.T, v4Enabled bool, v6Enabled bool) {
 		mockDataSet.EXPECT().AddRecord(gomock.AssignableToTypeOf(elemList), templateID).DoAndReturn(
```

Comment:
in a follow-up PR, please consider rewriting this test to use a custom gomock matcher: https://manadart.github.io/posts/20150212.html

Reply:
Sure. I will take a try to see whether I can include the change in this PR.